### DISCUSSION TITLE
Inverse Kinematics module parameters

### DISCUSSION THREAD

**Discussion by Ankitu7** [stack-of-tasks/pinocchio#2172] [Q&A]
I was wondering if the Inverse Kinematic module in your library will allow me to set parameters for the distance and orientation with respect to the origin and about what axis the joints/bodies of a 6DoF robot is able to rotate for C++.

**Comment by stephane-caron** [comment]
Can you link to the code or doc to make clear what inverse kinematic module are you referring to? 

AFAIK there are only IK examples in Pinocchio, not an IK module supported by the project. You can however check out other Pinocchio-based projects such as [TSID](https://github.com/stack-of-tasks/tsid) or [Pink](https://github.com/stephane-caron/pink/).
